### PR TITLE
VideoPress: Add option to load admin UI

### PR DESCRIPTION
**Pull request by leogermani**
#### Changes proposed in this Pull Request:

* Adds a way to choose whether we want to initialize the VideoPress admin UI or not, and initialize it from the package

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion

NA

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* Activate Jetpack and the VideoPress module
* Make sure you don't see the Jetpack > VideoPress menu in the admin menu
* Activate the VideoPress plugin
* Make sure you see the menu item now
* Deactivate Jetpack and confirm you still see the menu item

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202813985335436

### CODE REVIEW

File: projects/packages/config/src/class-config.php [hunk 1]
```diff
@@ -289,7 +289,17 @@ protected function enable_waf() {
 	 */
 	protected function enable_videopress() {
 		VideoPress_Pkg_Initializer::init();
+		return true;
+	}
 
+	/**
+	 * Hanldes VideoPress options
```
Comment (retrofox) [inline projects/packages/config/src/class-config.php]: minor: typo

```suggestion
	 * Handles VideoPress options
```

**Review by retrofox — COMMENTED** [review]
Tested and LGTM. I'm not too familiar with the implementation, though.  A couple of extra 👀 won't hurt at all.

- [x] Activate Jetpack and the VideoPress module
- [x] Make sure you don't see the Jetpack > VideoPress menu in the admin menu
image

- [x] Activate the VideoPress plugin
- [x] Make sure you see the menu item now

image

- [x]  Deactivate Jetpack and confirm you still see the menu item
image

**Comment by retrofox** [comment]
btw, thanks for the Readme file 💯

**Comment by retrofox** [comment]
Hi @zinigor, could you help us with this review in case you have the chance? Thanks in advance 🙇‍♂️ Personally I'm not too familiar with this area :-/

**Review by oskosk — DISMISSED** [review]
Tests well!

**Review by oskosk — APPROVED** [review]
Tests well!

**Pull request merged**